### ISSUE TITLE
Hover buttons

### ISSUE THREAD

**Issue by bertob** [nate-xyz/resonance#17]
Revealing controls on hover is generally not a good idea for a number of reasons, including discoverability, accessibility, and input method support (inaccessible via touch/keyboard). The app currently uses hover controls on album covers and track lists (and maybe others places I haven't encountered?), I'd revisit these and replace them with static buttons, context menus, or something else that doesn't require hover.

**Comment by nate-xyz** [comment]
These are good points, but I designed a lot of the UI around a cleaner look without play buttons everywhere in order to let album art take the main focus. However, I could definitely add settings to turn off hover controls. Would that solve your issue or do you think there should be no hover by default?

**Comment by BartleyTheShopkeeper** [comment]
![image](https://user-images.githubusercontent.com/73858626/230652429-7cf06fe5-3e66-405b-b0ae-b047b33061c4.png)
![image](https://user-images.githubusercontent.com/73858626/230652487-944cd0c7-60eb-4a8c-a12a-b4cc909ca4d0.png)

I'm no expert on development, nor on UI design. Maybe you could remove the hover-based buttons in favor of two buttons below the album's art on a given album's flap/page. The above screenshots are taken from Amberol. Those style buttons, but side by side (rather than stacked as seen above), are what I imagine could be used for your "play" and "queue" options.

Of course from the album gallery you might have a context menu that can play or add.

Just my two cents. I love resonance's concept.

**Comment by bertob** [comment]
> I could definitely add settings to turn off hover controls

That's generally a bad idea, the default UI should just work for most people.

FWIW there's plenty of clean-looking music apps that don't use hover in this way. I'd look at mobile/iPad apps for inspiration, where hover just isn't a thing.

**Comment by nate-xyz** [comment]
> I'd look at mobile/iPad apps for inspiration, where hover just isn't a thing.

I don't use iOS, could you give me a specific example?

**Comment by george-norton** [comment]
My laptop has a touch screen, obviously I cannot hover..
There appears to be a separate issue (maybe related?) where I am not able to click on albums with the touchscreen, scrolling works fine, but tapping on an album does nothing.

I am running Manjaro, Gnome and Wayland.

Edit: The first album I touch after starting resonance gets permanent hover buttons, they stay if I scroll with the touchscreen but disappear if I move the pointer around with the trackpad.
